### ISSUE TITLE
AutoSpotting logs too much by default

### ISSUE THREAD

**Issue by gabegorelick** [AutoSpotting/AutoSpotting#382]
## Issue type ##

Feature Idea

## Build number #

master

## Summary ##

AutoSpotting logs a lot of information by default. While it's great to have such more info should something go wrong, in practice I think it often makes it harder to debug issues. In an environment with a lot of auto scaling groups, the amount of logs can be overwhelming and makes it difficult to get a sense of what AutoSpotting is actually doing.

Some of the logs that are arguably not worth including by default:
- `Processing page`
- `Skipping group`

These can still be logged at the debug level, but I don't think they're worth logging at the default level.

## Steps to reproduce ##

Run AutoSpotting in a region with lots of ASGs.

## Expected results ##

AutoSpotting logs only what's needed to investigate issues that could arise.

## Actual results ##

So much output that it's hard to follow. I count 444 lines in my environment on an invocation where AutoSpotting didn't even launch or terminate any instances.

**Comment by cristim** [comment]
I agree with this, I  actually did a similar change on my work in progress branch for event based execution.
